src/plugins/intel_gpu/tests/test_cases/gather_gpu_test.cpp

```diff
@@ -8,10 +8,334 @@
 #include <intel_gpu/primitives/gather.hpp>
 
 #include <cstddef>
+#include <array>
 
 using namespace cldnn;
 using namespace ::tests;
 
+std::ostream& operator<<(std::ostream& os, FLOAT16 x) {
+    return os << float(x);
+}
+
+template <class T>
+int dim(const T& a) {
```

Comment:
what about changing name to `get_effective_dim` or `get_not_one_dim`?

Reply:
Done.